test/Microsoft.Dnx.Testing.SampleTests/DnuCommonUtilTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Dnx.Testing.SampleTests
+{
+    [Collection(nameof(SampleTestCollection))]
+    public class DnuCommonUtilTests : DnxSdkFunctionalTestBase
+    {
+        [Theory, TraceTest]
+        [MemberData(nameof(DnxSdks))]
+        public void FailTest(DnxSdk sdk)
+        {
+            var feedSolution = TestUtils.GetSolution<DnuCommonUtilTests>(sdk, "DependencyGraphsProject");
```

Comment:
Do you mean the CleanUpTestDir() function? If so, that was intentional since cleanup should, by default, only occur when the test passed. Failed tests should retain the files used during the test. I know it's kind of hacky but the only way I have right now of telling whether the test passed is if the last statement is reached. Unless I want to  extend the xUnit test runner class that is.